friends/src/actions/index.js

```diff
@@ -1,19 +1,55 @@
 import axios from 'axios';
+import axiosConfig from '../axiosConfig'
+// get friend
 
-export const FETCHING = 'FETCHING'
-export const SUCCESS = 'SUCCESS'
-export const FAILURE = 'FAILURE'
+export const FETCHING = 'FETCHING';
+export const SUCCESS = 'SUCCESS';
+export const FAILURE = 'FAILURE';
 
-const getFriends = () => dispatch => {
+// loging in
+export const LOGING_IN = 'LOGING_IN';
+export const PROCESSING_LOGIN = 'PROCESSING_LOGIN';
+export const LOGINGIN_FAILURE = 'LOGINGIN_FAILURE';
+
+// add new friend
+export const ADD_NEW_FRIEND_START = 'ADD_NEW_FRIEND_START';
+export const ADD_NEW_FRIEND_SUCCESS = 'ADD_NEW_FRIEND_SUCCESS';
+export const ADD_NEW_FRIEND_FAILURE = 'ADD_NEW_FRIEND_FAILURE';
+
+
+export const getFriends = () => dispatch => {
     dispatch({type: FETCHING })
```

Comment:
Try to make the name more detailed for each type of each axios call, like so: 'FETCHING_FRIENDS'